stfl/dialogs.stfl

```diff
@@ -13,7 +13,7 @@ vbox
   @bind_home[bind_home]:
   @bind_end[bind_end]:
   @on_TAB:"TAB"
-  label#title[title]
```

Comment:
We can't make these widgets nameless because `swap-title-and-hints` needs a way to manipulate them. Currently, the code from this PR segfaults if I enable that setting.

Reply:
I thought I checked these, but apparently missed this one.
Thanks!